src/IIIFPresentation/API/Features/Storage/StorageController.cs

```diff
@@ -50,9 +52,13 @@ public async Task<IActionResult> Get(int customerId, string id)
 
         if (storageRoot.Collection == null) return NotFound();
 
-        return Ok(Request.ShowExtraProperties()
-            ? storageRoot.Collection.ToFlatCollection(GetUrlRoots(), Settings.PageSize, storageRoot.Items)
-            : storageRoot.Collection.ToHierarchicalCollection(GetUrlRoots(), storageRoot.Items));
+        if (Request.ShowExtraProperties())
```

Comment:
Not directly related to changes in this PR but should this response include `Vary` header for x-iiif-show-extras?

Reply:
https://github.com/dlcs/iiif-presentation/issues/19